<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleBrakePage</class>
 <widget class="QWidget" name="VehicleBrakePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(0,0,50);
}
QLabel
{

font:24px,&quot;微软雅黑&quot;;
border:1px solid black;
}</string>
  </property>
  <widget class="QLabel" name="LBLM2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>60</y>
     <width>211</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLM1">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>211</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLMP1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>211</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLTC1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>211</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_brake1_1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/unknow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_brake1_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/unknow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_brake2_1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/unknow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_brake2_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/unknow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_brake3_1">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/unknow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_brake3_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/unknow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_brake4_1">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>70</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/unknow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_brake4_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>70</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/unknow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_metroNum2">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>25</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 170, 170);
font:20px,&quot;微软雅黑&quot;;
border:0px,black</string>
   </property>
   <property name="text">
    <string>MP1车</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_metroNum3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>25</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 170, 170);
font:20px,&quot;微软雅黑&quot;;
border:0px,black</string>
   </property>
   <property name="text">
    <string>MP2车</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_metroNum4">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>25</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 170, 170);
font:20px,&quot;微软雅黑&quot;;
border:0px,black</string>
   </property>
   <property name="text">
    <string>MC2车</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_metroNum1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>25</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 170, 170);
font:20px,&quot;微软雅黑&quot;;
border:0px,black</string>
   </property>
   <property name="text">
    <string>MC1车</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_stopbrake1">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>130</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/unknow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_stopbrake2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/unknow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_stopbrake3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>130</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/unknow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_stopbrake4">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>130</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/unknow.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cab2_direction">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>104</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/arrowright.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cab2_active">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>110</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 176, 80);
border-radius:8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cab1_direction">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>104</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/arrowleft.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cab1_active">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 176, 80);
border-radius:8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>60</y>
     <width>41</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border-top-left-radius:8px;
border-bottom-left-radius:8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>60</y>
     <width>41</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170,170,170);
border-top-right-radius:8px;
border-bottom-right-radius:8px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_break1_1_pressure">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>110</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px&quot;微软雅黑&quot;;
background-color: rgb(170, 170, 170);
border:0px;</string>
   </property>
   <property name="text">
    <string>1000Bar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_break1_2_pressure">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>110</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px&quot;微软雅黑&quot;;
background-color: rgb(170, 170, 170);
border:0px;</string>
   </property>
   <property name="text">
    <string>1000Bar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_break2_2_pressure">
   <property name="geometry">
    <rect>
     <x>434</x>
     <y>110</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px&quot;微软雅黑&quot;;
background-color: rgb(170, 170, 170);
border:0px;</string>
   </property>
   <property name="text">
    <string>1000Bar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_break2_1_pressure">
   <property name="geometry">
    <rect>
     <x>303</x>
     <y>110</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px&quot;微软雅黑&quot;;
background-color: rgb(170, 170, 170);
border:0px;</string>
   </property>
   <property name="text">
    <string>1000Bar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_break3_2_pressure">
   <property name="geometry">
    <rect>
     <x>644</x>
     <y>110</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px&quot;微软雅黑&quot;;
background-color: rgb(170, 170, 170);
border:0px;</string>
   </property>
   <property name="text">
    <string>1000Bar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_break3_1_pressure">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>110</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px&quot;微软雅黑&quot;;
background-color: rgb(170, 170, 170);
border:0px;</string>
   </property>
   <property name="text">
    <string>1000Bar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_break4_2_pressure">
   <property name="geometry">
    <rect>
     <x>854</x>
     <y>110</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px&quot;微软雅黑&quot;;
background-color: rgb(170, 170, 170);
border:0px;</string>
   </property>
   <property name="text">
    <string>1000Bar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_break4_1_pressure">
   <property name="geometry">
    <rect>
     <x>724</x>
     <y>110</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:18px&quot;微软雅黑&quot;;
background-color: rgb(170, 170, 170);
border:0px;</string>
   </property>
   <property name="text">
    <string>1000Bar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>LBLM2</zorder>
  <zorder>LBLM1</zorder>
  <zorder>LBLMP1</zorder>
  <zorder>LBLTC1</zorder>
  <zorder>lbl_brake1_1</zorder>
  <zorder>lbl_brake1_2</zorder>
  <zorder>lbl_brake2_1</zorder>
  <zorder>lbl_brake2_2</zorder>
  <zorder>lbl_brake3_1</zorder>
  <zorder>lbl_brake3_2</zorder>
  <zorder>lbl_brake4_1</zorder>
  <zorder>lbl_brake4_2</zorder>
  <zorder>lbl_metroNum2</zorder>
  <zorder>lbl_metroNum3</zorder>
  <zorder>lbl_metroNum4</zorder>
  <zorder>lbl_metroNum1</zorder>
  <zorder>lbl_stopbrake1</zorder>
  <zorder>lbl_stopbrake2</zorder>
  <zorder>lbl_stopbrake3</zorder>
  <zorder>lbl_stopbrake4</zorder>
  <zorder>lbl_cab2_direction</zorder>
  <zorder>lbl_cab2_active</zorder>
  <zorder>lbl_cab1_direction</zorder>
  <zorder>lbl_cab1_active</zorder>
  <zorder>lbl_break1_1_pressure</zorder>
  <zorder>lbl_break1_2_pressure</zorder>
  <zorder>lbl_break2_2_pressure</zorder>
  <zorder>lbl_break2_1_pressure</zorder>
  <zorder>lbl_break3_2_pressure</zorder>
  <zorder>lbl_break3_1_pressure</zorder>
  <zorder>lbl_break4_2_pressure</zorder>
  <zorder>lbl_break4_1_pressure</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
